library/HTMLPurifier/HTMLModule/HTML5/Link.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * https://html.spec.whatwg.org/dev/semantics.html#the-link-element
+ */
+class HTMLPurifier_HTMLModule_HTML5_Link extends HTMLPurifier_HTMLModule
```

Comment:
This module should definitely be marked as _unsafe_.

Reply:
Will move to unsafe list per https://github.com/xemlock/htmlpurifier-html5/pull/63#discussion_r634749873